zipkin-ui/js/page/default.js

```diff
@@ -27,12 +27,14 @@ const DefaultPageComponent = component(function DefaultPage() {
       const limit = query.limit || this.attr.config('queryLimit');
       const minDuration = query.minDuration;
       const endTs = query.endTs || new Date().getTime();
+      const startTs = query.startTs || endTs - this.attr.config('defaultLookback');
```

Comment:
I guess it's not necessary (just a suggestion for readability), but a set of parentheses here would help clarify, in case you cannot remember operator precendence between || and - :wink:

```javascript
query.startTs || (endTs - this.attr.config('defaultLookback'));
```